src/components/Card.js

```diff
@@ -58,11 +75,17 @@ export default class Card {
   #completeNewCard () {
     this.#card = this.#returnEmptyClone();
     this.#likeButton = this.#card.querySelector(".card__like-button");
+    this.#likeCounter = this.#card.querySelector(".card__like-count");
```

Comment:
Good job that you've found these variables only once here and defined them as **class fields**